src/WatchDogs.Infrastructure.FakeSource/FakeTradeGenerator.cs

```diff
@@ -20,6 +22,9 @@ public class FakeTradeGenerator : IFakeTradeGenerator
 
     Faker<Trade> tradeModelFake;
 
+    private readonly ApplicationDbContext _context;
```

Comment:
What is the meaning behind these fields? I don' see any use of their functionality.

Reply:
The purpose of those fields was lost down the road and remains mystery to these days